clinical-domain-agent/src/main/java/care/smith/fts/cda/DefaultTransferProcessRunner.java

```diff
@@ -51,19 +127,25 @@ public class TransferProcessInstance {
     private final TransferProcessDefinition process;
 
     private final AtomicReference<TransferProcessStatus> status;
+    private final String processId;
```

Comment:
the processId is also in the status. So it is redundant. I would remove it from one of the locations.